evm_loader/program/src/executor/precompile_extension/metaplex.rs

```diff
@@ -50,9 +50,9 @@ pub fn metaplex<B: AccountStorage>(
             }
 
             let mint = read_pubkey(input);
-            let name = read_string(input, 32);
-            let symbol = read_string(input, 64);
-            let uri = read_string(input, 96);
+            let name = read_string(input, 32)?;
```

Comment:
What is invalid input for `read_pubkey`? I know it is only an invalid input length, but we get a fault in this case.

Reply:
Panic will crash a solana transaction and cause an entire ethereum transaction to fail.
Returning an error will cause only one call to precompile to fail, and this can be handled by the Solidity contract.

Reply:
Fixed